clang/include/clang/Basic/Attr.td

```diff
@@ -1239,6 +1239,23 @@ def SYCLSimdAccessorPtr : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+// The attribute denotes that it is a function written in SIMT fashion, which
+// is used in ESIMD context and needs to be widened by a vector backend compiler.
+// For now, this attribute will be used only in internal implementation of
+// specific public ESIMD APIs. It is not supposed to be used directly in the
+// user code, hence it is undocumented.
+// The argument of the attribute specifies the number of SIMD lanes, for which
+// the function should be widened.
+def SYCLIntelESimdWiden : InheritableAttr {
+  let Spellings = [GNU<"sycl_esimd_widen">, CXX11<"intel", "sycl_esimd_widen">];
```

Comment:
I'm ok with just leaving a GNU-style, but is there a downside of supporting both styles?

Reply:
No major downside outside of the usual annoyance with GNU-style attributes being complex to use (my preference is to use `[[]]` spellings because it's a SYCL-related attribute and SYCL requires C++, so I don't see why the GNU spelling would ever be needed).

Reply:
That makes sense. Will update.